packages/forma-36-website/src/pages/components/card/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Card'
+type: 'component'
+status: 'stable'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Card'
+storybook: https://f36-storybook.contentful.com/?path=/story/components-card-entrycard--default'
+---
+Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
+
+```jsx
+<React.Fragment>
+  <div>
+    <EntryCard
```

Comment:
This should just be the `<Card>` component

Reply:
@m10l yeah, I thought it would just look better for now. We would probably add here documentation for all type of Cards anyway. but no prob, I can changed it to the simple card.

Reply:
And done :)